README.md

```diff
@@ -98,6 +98,21 @@ Convert an observation to an OSM type `node`.
 
 Retrieves the current device's unique ID (string).
 
+### `mapeo.getDatabaseStatus(cb)`
+
+Retrieves the status of the database, returns an array of Device IDs and their download status.
+
+```
+{ 
+  devices: [{
```

Comment:
At some point we might want to add support for media items too in this API, so I think we should prefix the sofar and total with something e.g. `dbTotal`. We could also nest this e.g. `db.total` but nested structures are always a pain to deal with in JS just because of the risk of `undefined` props throwing an error.